osgp/platform/osgp-adapter-kafka-distributionautomation/pom.xml

```diff
@@ -176,6 +176,14 @@
       <groupId>org.springframework.kafka</groupId>
       <artifactId>spring-kafka-test</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-reflect</artifactId>
+    </dependency>
```

Comment:
Shouldn't these be testing dependencies?

Reply:
Changed that.